openapi-cli/src/main/resources/cli-help/ballerina-openapi-add.help

```diff
@@ -0,0 +1,72 @@
+NAME
+       bal openapi add - Update the Ballerina.toml file with the OpenAPI tool configuration details for generating a Ballerina service or client.
+
+SYNOPSIS
+       bal openapi add [-i | --input] <openapi-contract-file-path>
+                       [-p | --package] <package-location>
+                       [--id] <service/client id>
+                       [--module] <module-name-for-generated-service/client>
+                       [--mode] <mode-type>
+                       [--tags] <tag-names>
+                       [--operations] <operation-names>
+                       [-n | --nullable]
+                       [--license] <license-file-path>
+                       [--client-methods] <resource|remote>
+
+DESCRIPTION
+       Update the Ballerina.toml file with the OpenAPI tool configuration details for generating a Ballerina source
+       (either a mock service or a client stub) from the given OpenAPI definition file and other command options.
+
+OPTIONS
+       -i, --input <openapi-contract-file-path>
+            This is a mandatory input. The given OpenAPI contract input will
+            generate a Ballerina service and the client stub files for the given
+            OpenAPI contract. The OpenAPI contract can be either a YAML or a JSON
+
+       --id <service/client id>
+            This is a mandatory input. The given service/client id will be used
+            to identify the generated service/client in the Ballerina.toml file.
+            This id will be used to generate the Ballerina service/client
+            sources cache in the target.
+
+       --module <module-name-for-generated-service/client>
+            This is a mandatory input. The given module name will be used to
```

Comment:
I think in the demo when we don't have this value the files are generated in the `generated` directory right? Did we fix that in the generation path(via `bal build`)? I think if we generate the files in the `generated` directory then we can not import them in the main module

Reply:
Yeah, I assume they provide a default path as generated directory, will check on the bal build implementation.